src/coprocessor/statistics/analyze.rs

```diff
@@ -96,6 +101,13 @@ impl<S: Snapshot> AnalyzeContext<S> {
 
         let mut row_count = 0;
         let mut time_slice_start = Instant::now();
+        let mut topn_heap = BinaryHeap::new();
+        let mut cur_v: (i32, Vec<u8>) = (0, Vec::from(""));
+        let top_n_size = req.get_top_n_size() as usize;
+        let mut stats_version = VERSION1;
+        if req.has_version() {
+            stats_version = req.get_version();
+        }
```

Comment:
```suggestion
        let stats_version = if req.has_version() { req.get_version() } else VERSION1;
```

Reply:
`VERSION1` isn't a clear name. How about `ANALYZE_INDEX_REQ_V1`?

Reply:
Oh this will version will also affect analyze column in the future. Or just `ANALYZE_VERSION_V1`?

Reply:
sure